pkg/cli/debug/debug.go

```diff
@@ -1063,7 +1061,19 @@ func (o *DebugOptions) approximatePodTemplateForObject(object runtime.Object) (*
 					},
 				},
 			},
-		}, nil
+		}
+		if t.Labels[corev1.LabelOSStable] == "windows" {
+			template.Spec.OS = &corev1.PodOS{Name: corev1.Windows}
+			template.Spec.HostPID = false
+			template.Spec.HostIPC = false
+			containerUser := "ContainerUser"
```

Comment:
From where this user comes from and why it is static?

Reply:
There are only two possible users for Windows containers, ContainerUser and ContainerAdministrator.
https://learn.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/container-security#when-to-use-containeradmin-and-containeruser-user-accounts

ContainerAdministrator would give the debug container full root access to the VM, so I'd think ContainerUser would be preferred.

Reply:
Good idea to use `ContainerUser` instead admin.